### ISSUE TITLE
Extra tabbing is necessary when using a screen reader.

### ISSUE THREAD

**Issue by TheQuinbox** [UnigramDev/Unigram#2767]
**Describe the bug**
There are a lot of extra controls in the Unigram window that I, as a screen reader user, have to tab between. Takes half a second or so normally, but over time this can add up.
**To Reproduce**
1. Launch the NVDA screen reader.
2. Open Unigram
3. attempt to tab between all the controls.
**Expected behavior**
I should be able to get directly to my chats list from the tab control, without having to press tab. I should also be able to get off of it without pressing tab twice. In addition, I have no idea what this "Nothing to see here, tab further" button is, but it doesn't seem to do anything accessible, and is also just extra tabbing.
**Version Info**
 - Unigram 8.8.7433.0
 - Windows: Windows 10 21H2 (x64) build 19044.1706

**Comment by Elshara** [comment]
Yeah, I can confirm this as well.

**Comment by Kuchizu** [comment]
@Elshara Ok

**Comment by MarcAnt01** [comment]
@TheQuinbox hello, thanks for opening the ticket, anyway we need the exact version number, not "LATEST". You can find the version number in settings

**Comment by Elshara** [comment]
I experience the same in Unigram_8.8.7433.0

**Comment by MarcAnt01** [comment]
Thanks, I've added this version to the ticket

**Comment by amirmahdifard** [comment]
this is already fixed as all of the silence tabs have been removed and the nothing to see here is removed as whel.
